Ship_Game/Universe/UniverseState.cs

```diff
@@ -615,13 +615,13 @@ public void OnPlanetOwnerAdded(Empire owner, Planet planet)
             if (planet.Budget?.Owner != owner)
                 planet.CreatePlanetBudget(owner);
             owner!.AddBorderNode(planet);
-            Influence.Insert(owner, planet);
+            Influence.Insert(owner, planet.System);
```

Comment:
This introduces a massive bug, previously a planet was carefully checked so it would only be added once.
And so, there is no protection to adding multiple influences for the same entity in Influence tree Insert. You can check the Insert() function and can see that it simply inserts the object without any protective checks.

So you should add protection inside the Insert() - which shouldn't be an issue for performance tbh. You can use debug visualizer to see if multiple influences are added.